main/gdk-pixbuf/APKBUILD

```diff
@@ -1,22 +1,20 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gdk-pixbuf
 pkgver=2.36.11
-pkgrel=2
+pkgrel=3
 pkgdesc="GTK+ image loading library"
 url="https://www.gtk.org/"
 arch="all"
-license="LGPL-2.0+"
-depends="shared-mime-info"
-depends_dev=""
+license="LGPL-2.0-or-later"
+depends="shared-mime-info librsvg"
```

Comment:
@ncopa is the addition of librsvg alright ? it is needed for programs that want to load svg files with gdk-pixbuf-loader, like dunst which uses it in its tests.

if it is added here librsvg it won't be necessary to add librsvg on a case by case basis but it will increase the dependencies of gdk-pixbuf.